systems/registry/member/github.com/ukama/ukama/systems/registry/network/pb/gen/network.pb.go

```diff
@@ -0,0 +1,1535 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
We need to fix the  makefile proto gen path so that generated files are place in the the right pb/ sub-folder  instead of created a new github.com/... tree.

Reply:
removed